app/static/js/yara_rule/yara_rule-controller.js

```diff
@@ -821,7 +824,13 @@ angular.module('ThreatKB')
 
             $scope.cfg_states = Cfg_states.query();
             $scope.cfg_category_range_mapping = CfgCategoryRangeMapping.query();
-            $scope.do_not_bump_revision = true;
+            if ($scope.do_not_bump_revision == null) {
+                $scope.do_not_bump_revision = true;
+            }
+
+            $scope.toggle_bump_release = function () {
+                $scope.do_not_bump_revision = this.do_not_bump_revision;
+            };
```

Comment:
Came across a bug while working. The `do_not_bump_revision` was not working as expected. It was ALWAYS bumping irrespective of whether it was checked or not.